src/adams_utils.jl

```diff
@@ -64,5 +64,10 @@ function g_coefs!(cache, k)
       end # q
       g[i] = c[i,1]
     end # i
-  end # inbounds
+  # end # inbounds
+end
+
+function γstar(k)
+  γ_star = [1,-1/2,-1/12,-1/24,-19/720,-3/160,-863/60480,-275/24192,-33953/3628800]
```

Comment:
This is expensive. Just store the global array as a constant and access it.

Reply:
okay

Reply:
Do we care about the precision of these constants? This will just always be a `Float64` array this way, but could we ever need to worry about having to change the precision on this (and do the converting Rational tricks instead)?